python/docs/source/getting_started/installation.rst

```diff
@@ -0,0 +1,84 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+============
+Installation
+============
+
+Using PyPI
+~~~~~~~~~~
+PySpark installation using `PyPI <https://pypi.org/project/pyspark/>`_::
+
+    pip install pyspark
+	
+Using Miniconda  
+~~~~~~~~~~~~~~~
+Pyspark is available in Conda but it is not a part of an official release. Conda can be used to create a virtual environment from terminal as shown below::
+
+	conda create -n ENV_NAME python==PYTHON_VERSION
+
+After the virtual environment is created, it should be visible under the list of conda environments which can be seen using the following command::
+
+	conda env list
+
+The newly created environment can be accessed using the following command::
+
+	source activate ENV_NAME
+
+On Windows, the command is::
+
+	conda activate ENV_NAME
+
+PySpark can be installed in this newly created environment using PyPI as shown before::
+
+	pip install pyspark
+
+Official release channel
```

Comment:
I think it should be `Official Release Channel` to be consistent with other title styles in Spark documentation.